api/nakadi-event-bus-api.yaml

```diff
@@ -1202,16 +1202,12 @@ definitions:
 
       default_statistics:
         type: object
+        allOf:
```

Comment:
There are 2 reasons.
 - While using different ui generators description field on `EventTypeStatistics` will be used itself, and I do not want it (because someday there will be measured_statistics).
 - Default statistics can hold some additional information.